templates/registration/register.html.twig

```diff
@@ -0,0 +1,77 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Inscription{% endblock %}
+
+{% block body %}
+    {% include "components/_header.html.twig" %}
+    {% for flash_error in app.flashes('verify_email_error') %}
```

Comment:
same here, I think that the flashes error message display should be in the base.html.twig, bellow the header